packages/pwa-kit-create-app/scripts/trim-extensions.js

```diff
@@ -120,91 +130,104 @@ function processFile(filePath, plugins) {
         }
     }
 
+    const nodeContainsPluginReferences = (node) => {
+        return pluginRegex.test(source.slice(node.start, node.end))
+    }
```

Comment:
Additional method to find out if a code snippet (node) contains one of the plugin flags.